### PR TITLE
> fix typos

### PR DESCRIPTION
**Pull request by yowenter**

### CODE REVIEW

File: sig-cluster-lifecycle/postmortems/kubeadm-1.6.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ initialize a new cluster with the 1.6.0 release
 
 - There was a lot of rushing to get the release ready before KubeCon EU, causing
   a shortened timeframe for RC and release, lowered bandwidth for SIG members,
-  and the last SIG meeting before the release to be cancelled, which decreased
+  and the last SIG meeting before the release to be canceled, which decreased
```
Comment (cblecker) [inline sig-cluster-lifecycle/postmortems/kubeadm-1.6.md]: This is the British spelling, and can be removed from this PR.

Comment (yowenter) [reply sig-cluster-lifecycle/postmortems/kubeadm-1.6.md]: 👌， thanks for your tips

**Comment by cblecker** [comment]
@yowenter Oh, also, please remove the changes to `events/2016/developer-summit-2016/`. While the spelling corrections you've made are correct, these directories are historical and not intended to be changed. Thanks!

**Comment by cblecker** [comment]
/ok-to-test
/lgtm
/approve

Thanks for the PR!

**Pull request merged**